site/docs/v1/tech/apis/hosted-backend.adoc

```diff
@@ -0,0 +1,400 @@
+
+|500
+|There was a FusionAuth internal error. A stack trace is provided and logged in the FusionAuth log files.
+|===
+
+== Refresh
+
+This endpoint will extract the `app.rt` cookie if present and use it to make a `refresh_token` request from `/oauth/token`. If successful a new set of cookies will be set on the response the will continue to allow access to the application.
+
+This is an api that should be made from the client application and the browser should *NOT* be redirected to this endpoint.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#jwt-authentication[icon:id-badge[type=fas]] Refresh the current session cookies.
+
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/app/refresh/{clientId}#
+--
+
+==== Request Parameters
+
+[.api]
+[field]#client_id# [type]#[UUID]# [required]#Required#::
+The top-level client Id for your application.
+
+.Example Request URL
+[source,url]
+----
+https://auth.yourdomain.io/app/refresh/297ca84b-69a9-4508-8649-97644e1d0b3d
+----
+
+=== Response
+
+A successful response should set cookies and return a `200`.
+
+[cols="1a,9a"]
+.Response Codes
+|===
+|Code |Description
+
+|200
+|This is likely an error case. The route will serve up an error page with html and details on what went wrong.
+
+|400
+|The request was not successful. The client may need to reauthorize.
+
+|500
+|There was a FusionAuth internal error. A stack trace is provided and logged in the FusionAuth log files.
+|===
+
+== Me
+
+This api is used to retrieve information about the currently logged in user. This call will take the `app.at` cookie value and use that to call the `/oauth2/userinfo` API.
+
+This is an api that should be made from the client application and the browser should *NOT* be redirected to this endpoint.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#jwt-authentication[icon:id-badge[type=fas]] Get info about the user.
+
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/app/me/#
+--
+
+.Example Request URL
+[source,url]
+----
+https://auth.yourdomain.io/app/me
+----
+
+=== Response
+
+A successful response should set cookies and return a `302` redirect to the `redirect_uri` specified in the initial request. Other status codes indicate an error.
+
+[cols="1a,9a"]
+.Response Codes
+|===
+|Code |Description
+
+|200
+|The request was successful. The response will contain a JSON body.
+
+|401
+|The user is not authorized. May need to refresh or log in again.
+
+|500
+|There was a FusionAuth internal error. A stack trace is provided and logged in the FusionAuth log files.
+|===
+
+==== Response Body
+
+include::docs/v1/tech/shared/_user-info-response.adoc[]
+
+== Logout
+
+This API will start a logout. The cookies specific to this session will be removed and if an SSO session was started it will be ended.
+
+To use this API correctly the browser should be redirected to this route and it will respond with a `302` redirect status code if successful.
```

Comment:
```suggestion
To use this API, redirect the browser to this route. This is not meant to be called by non-browser clients.
```

No need to mention the 'if successful' since that is handled by the response table.